demo/app-root.ts

```diff
@@ -259,7 +310,7 @@ export class AppRoot extends LitElement {
   private async fetchSearchResults(query: string, searchType: SearchType) {
     const sortParam =
       this.checkedSort?.value === 'none'
-        ? []
+        ? undefined
```

Comment:
Q: does this change behavior?

Reply:
Just prevents adding an unnecessary `&sort=` blank param if there is no sort option selected.